pkg/printer/workload_status_printer_test.go

```diff
@@ -0,0 +1,299 @@
+/*
```

Comment:
license format is wrong. can you copy paste it the way its done in other files

Reply:
Missed this, thank you.